site/examples/paste-html.js

```diff
@@ -58,10 +58,7 @@ export const deserialize = el => {
   ) {
     parent = el.childNodes[0]
   }
-
-  const children = Array.from(parent.childNodes)
-    .map(deserialize)
-    .flat()
+  const children = [].push(...Array.from(parent.childNodes).map(deserialize))
```

Comment:
I'd recommend instead using a polyfill to incorporate the flat call for legacy edge

Reply:
Following up on this, if we went with the polyfill, it'd need to be scoped to the dependencies of the example site, not the slate packages.

Reply:
@CameronAckermanSEL thanks for the feedback!
Got it. I have removed local support of flat. Let me know if anything else needs attention.